src/t8_cmesh/t8_cmesh.cxx

```diff
@@ -1245,7 +1242,7 @@ t8_cmesh_reset (t8_cmesh_t *pcmesh)
      * This is useful for debugging. */
     if (t8_cmesh_is_committed (cmesh)) {
       comm = t8_shmem_array_get_comm (cmesh->tree_offsets);
-      T8_ASSERT (t8_cmesh_comm_is_valid (cmesh, comm));
+      T8_ASSERT (t8_cmesh_get_mpicomm (cmesh) == comm);
```

Comment:
I would still use `t8_cmesh_comm_is_valid ` here because of the straighforward naming
```suggestion
      T8_ASSERT (t8_cmesh_comm_is_valid (cmesh, comm);
```